packages/core/src/common/_mixins.scss

```diff
@@ -103,6 +103,39 @@ $pt-dark-intent-text-colors: (
   @include force-hardware-acceleration();
 }
 
+// A collection of mixins that utilize the CSS Flexbox model to standardize
+// the layout and composition of Blueprint components.
+
+// pt-flex-parent styles
+@mixin pt-flex-parent() {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+
+@mixin pt-flex-parent-fill() {
```

Comment:
this mixin really has nothing to do with flex and it's an unnecessary level of abstraction.
let's just remove it and write the one line of CSS where necessary.

Reply:
@giladgray seems reasonable